app/models/invenio_rdm_record_converter.rb

```diff
@@ -109,14 +112,15 @@ def invenio_provenance(proxy_depositor, on_behalf_of)
   def invenio_metadata(gf)
     {
       "resource_type": resource_type(gf.resource_type.shift),
-      "creators": creators(gf.creator),
+      "creators": gf.creator.map{ |creator| build_creator_contributor_json(creator) },
```

Comment:
Refactoring the original creators function into #build_creator_contributor_json(creator) allowed for a simple map to be called like this.